recipes/tinytimer/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: tinytimer
+  version: "0.0.0"
+
+source:
+  fn: tinytimer-0.0.0.tar.gz
+  url: https://files.pythonhosted.org/packages/5c/87/75d3fbe15aabd45d9b70702241787cf1f7f30dd9fabcd9bc89d828c7661d/tinytimer-0.0.0.tar.gz
```

Comment:
use canoncial url and `sha256`. See the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml